.github/workflows/swagger.yml

```diff
@@ -0,0 +1,37 @@
+name: Open API Specification and Documentation
+on: [push, pull_request]
+jobs:
+  swaggerci:
+    name: swagger-build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup go-swagger
+        uses: minchao/setup-go-swagger@v1
+        with:
+          version: v0.26.1
+
+  build:
+    needs: [swaggerci]
+    name: Swagger-ci
+    runs-on: ubuntu-latest
+    steps:
+    - name: swagger-build
+      uses: actions/checkout@v2
+      run: |
+          sudo apt install snapd
+          sudo snap install go-swagger
+          sudo apt update
+    - name: Swagger-Workflow
+      uses: actions/cache@v1
+      with:

```

Comment:
Remove these above 3 lines. They are not needed. I think we don't need to save cache yet.

Reply:
So, besides we don't need them, they are badly indented. For that reason the workflow is failing